test/tbbmalloc/test_malloc_regression.cpp

```diff
@@ -158,13 +158,16 @@ void TestAlignedMsize()
         }
 }
 
-//! \brief \ref error_guessing
+#if __TBB_USE_ADDRESS_SANITIZER
+TEST_CASE("Memory leaks test is not applicable under ASAN\n" * doctest::skip(true)) {}
+#else
```

Comment:
Do not remove the briefs. In addition, each TEST_CASE (incl. skipped) should have a brief. Maybe it make sense just to exclude the TEST_CASE, e.g.
```suggestion
#if !__TBB_USE_ADDRESS_SANITIZER
//! \brief \ref error_guessing
```